combirepo/rpm_patcher.py

```diff
@@ -691,11 +703,11 @@ def __use_cached_root_or_prepare(self):
         else:
             self.__prepare()
             cached_chroot_path = os.path.join(
-                patching_cache_path, os.path.basename(
-                    self.patching_root) + "preliminary_image")
+            patching_cache_path, os.path.basename(
+                self.patching_root) + "preliminary_image")
             hidden_subprocess.call(
                 "Saving chroot to cache",
-                ["sudo", "cp", "-a", self.patching_root, cached_chroot_path])
+                ["sudo", "cp", "-P", "-a", self.patching_root, cached_chroot_path])
```

Comment:
I think we'd better add `-Z` flag as well, so SMACK attributes will be copied. This might be useful in certain cases.

Reply:
Done